docs/authentication.md

```diff
@@ -0,0 +1,75 @@
+## Tock Authentication
+
+[:arrow_left: Back to Tock
+Documentation](https://github.com/18F/tock/tree/master/docs)
+
+18F's current deployment of Tock relies on
+[cloud.gov's User Account and Authentication (UAA) server][UAA] for
+authentication.
+
+During development, a ["fake" cloud.gov UAA server][fakeUAA] is used for
+authentication. Here you can actually enter any email address; if the
+address is `@gsa.gov`, then a non-staff account will automatically
+be created for the user and you'll be logged-in, but otherwise access
+will be denied.
+
+The easiest way to create an administrative user is to first use
+`manage.py createsuperuser` to create a user, and then log in
+with that user's email address.  See the "Getting Started" section
+for an example of this.
```

Comment:
It feels like before we say to use `manage.py` we should add a line about how to get to it from within docker. I know that tripped me up. But that may just be me.